Installation/include/CGAL/config.h

```diff
@@ -466,9 +466,9 @@ using std::max;
 
 //-------------------------------------------------------------------//
 // Is Geomview usable ?
-#if !defined(_MSC_VER) && !defined(__MINGW32__)
-#  define CGAL_USE_GEOMVIEW
-#endif
+//#if !defined(_MSC_VER) && !defined(__MINGW32__)
+//#  define CGAL_USE_GEOMVIEW
```

Comment:
intermediate patch that should be undo?

Reply:
no, it's more of a quick and easy way of disabling Geomview without modifying 30 files and preventing the testing of several that conditionally include geomview files.